meson.build

```diff
@@ -0,0 +1,73 @@
+project('eos-event-recorder-daemon',
+        'c',
+        license: 'GPL-2.0-or-later',
+        version: '0.0.0',
+        default_options: ['c_std=c11'],
+        meson_version: '>= 0.52.0',
+)
+
+cc = meson.get_compiler('c')
+
+eosmetrics_dep = dependency('eosmetrics-0', version: '>= 0.2')
+polkit_gobject_dep = dependency('polkit-gobject-1')
+
+emer_shared_required_modules = [
+    dependency('gio-unix-2.0', version: '>= 2.46'),
+    dependency('gobject-2.0', version: '>= 2.58'),
+    dependency('sqlite3'),
+    dependency('uuid'),
+]
+emer_required_modules = emer_shared_required_modules + [
+    eosmetrics_dep,
+    cc.find_library('m', required: false),
+    dependency('glib-2.0', version: '>= 2.63.1'),
+    dependency('ostree-1', version: '>= 2013.7'),
+    polkit_gobject_dep,
+    dependency('libsoup-2.4', version: '>= 2.42'),
+]
+
+gnome = import('gnome')
+py = import('python').find_installation('python3',
+    modules: [
+        # Meson can't express dbusmock >= 0.10 here
+        'dbusmock',
+        'dbus',
+    ],
+)
+
+systemd_dep = dependency('systemd')
+event_recorder_server_xml = eosmetrics_dep.get_pkgconfig_variable('datadir') / 'eos-metrics' / 'com.endlessm.Metrics.xml'
+
+default_metrics_server = get_option('default_metrics_server')
+
+prefix = get_option('prefix')
+libexec_dir = join_paths(prefix, get_option('libexecdir'))
+# TODO: remove trailing slash, inherited from Autotools build
```

Comment:
Is this `TODO` comment something you want to address before this MR is merged?

Reply:
Nope, I was going for zero source changes.

Reply:
That makes sense!

Reply:
It's not actively harmful, it's just not to my taste :)